tests/Alcaeus/MongoDbAdapter/MongoCollectionTest.php

```diff
@@ -630,6 +889,39 @@ public function testGroup()
         );
     }
 
+    // This test won't work on travis should investigate later...
```

Comment:
@alcaeus here you go https://travis-ci.org/alcaeus/mongo-php-adapter/jobs/102574575